File: SoftInvPersistance/src/main/java/pe/edu/pucp/softinv/dao/EmpleadoDAO.java
package pe.edu.pucp.softinv.dao;

import pe.edu.pucp.softinv.model.Personas.EmpleadoDTO;

import java.util.List;

public interface EmpleadoDAO {
    int insertar(EmpleadoDTO empleado);
    int modificar(EmpleadoDTO empleado);
    int eliminar(int empleadoId);
    EmpleadoDTO obtenerPorId(int empleadoId);
    List<EmpleadoDTO> listarTodos();
}

File: C#/SoftInvPersistance/DAO/EmpleadoDAO.cs
using softinv.model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoftInv.DAO
{
    public interface EmpleadoDAO
    {
        int insertar(EmpleadoDTO empleado);
        int modificar(EmpleadoDTO empleado);
        int eliminar(int empleadoId);
        EmpleadoDTO obtenerPorId(int empleadoId);
        IList<EmpleadoDTO> listarTodos();
    }
}
